### ISSUE TITLE
alicloud_cs_managed_kubernetes: No data disk created for worker_data_disks block

### ISSUE THREAD

**Issue by embik** [aliyun/terraform-provider-alicloud#2684]
### Terraform Version
```
$ terraform -version
Terraform v0.12.24
```

Provider version is **v1.92.0**.

### Affected Resource(s)
- `alicloud_cs_managed_kubernetes`
- `alicloud_cs_kubernetes` (possibly, not verified)

### Terraform Configuration Files
```hcl
variable "ack_cluster_addons" {
  description = "The block to pass for Aliyun cluster addons. Be sure to have terway-eniip in there to get Terway support."
  type = list(object({
    name   = string
    config = string
  }))

  default = [
    {
      "name"   = "terway-eniip",
      "config" = "",
    },
    {
      "name"   = "flexvolume",
      "config" = "",
    },
    {
      "name"   = "alicloud-disk-controller",
      "config" = "",
    }
  ]
}

variable "ack_service_cidr" {
  description = "The Service CIDR that will be used by the ACK cluster."
  type        = string
  default     = "172.21.0.0/20"
}

resource "alicloud_cs_managed_kubernetes" "ack" {
  name_prefix                 = "marvin-ack"
  version                     = "1.16.9-aliyun.1"
  availability_zone           = "cn-zhangjiakou-c"
  worker_vswitch_ids          = [alicloud_vswitch.worker_nodes.id]
  pod_vswitch_ids             = [alicloud_vswitch.pods.id]
  new_nat_gateway             = true
  worker_instance_types       = ["ecs.g6.xlarge"]
  worker_number               = 4
  worker_instance_charge_type = "PostPaid"
  enable_ssh                  = true
  install_cloud_monitor       = false
  cpu_policy                  = "none"
  proxy_mode                  = "iptables"
  node_cidr_mask              = 25
  key_name                    = "marvin-key"
  service_cidr                = var.ack_service_cidr
  slb_internet_enabled        = true
  worker_disk_category        = "cloud_ssd"
  worker_disk_size            = 128
  kube_config                 = "${path.root}/kubeconfig"

  service_account_issuer = "kubernetes.default.svc"
  api_audiences          = ["kubernetes.default.svc"]

  dynamic "addons" {
    for_each = var.ack_cluster_addons
    content {
      name   = lookup(addons.value, "name", var.ack_cluster_addons)
      config = lookup(addons.value, "config", var.ack_cluster_addons)
    }
  }

  worker_data_disks {
    category  = "cloud_ssd"
    size      = 128
    encrypted = false
  }

}

```

### Debug Output

Unable to share at this time, sorry. I think this should be 100% reproducible, but if it's not please let me know and I will try to provide a debug output.

### Panic Output
/

### Expected Behavior

ACK nodes should have two disks attached, one system disk with 128 GB capacity, one data disk with 128 GB capacity (both of type cloud_ssd).

### Actual Behavior

Nodes came up with only the 128 GB **system** disk, no further disk was attached. Here's a screen capture of a node created by my cluster in the ECS console:

![Screenshot 2020-08-03 at 09 57 14](https://user-images.githubusercontent.com/10295525/89159523-e3e8ea80-d56f-11ea-976e-5931e49cc0c6.png)

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

Nothing that I am aware of.

### References

/

**Comment by embik** [comment]
Hi @xiaozhu36, sorry for bothering you, but could you take a look at this? We're wondering if we're doing something wrong or if this is a bug you can reproduce.

**Comment by demonwy** [comment]
hi @embik  thanks for your kind report. we will fix it next week.

**Comment by embik** [comment]
Thank you @demonwy, that is great news and much appreciated. Looking forward to any fixes since I was unable to determine the reason for this myself, otherwise I would have sent a PR to fix it.

**Comment by demonwy** [comment]
hi @embik this issue has been fixed on release 1.96.0, please check it

**Comment by embik** [comment]
Hi @demonwy, I've tested this with 1.98.0 and can confirm it works now! Great news. I was following the release changelog and didn't see a fix in there, but this seems to be working properly now. Thank you very much!
